src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -311,6 +317,40 @@ public static ScriptHost Create(ScriptHostConfiguration scriptConfig = null)
             return scriptHost;
         }
 
+        private static void LoadExtensions(ScriptHostConfiguration scriptConfig)
+        {
+            // Register our built in extensions
+            var hostConfig = scriptConfig.HostConfig;
+            scriptConfig.BindingProviders = new Collection<ScriptBindingProvider>()
+            {
+                new WebJobsCoreScriptBindingProvider(hostConfig),
+                new ServiceBusScriptBindingProvider(hostConfig),
+                new CoreExtensionsScriptBindingProvider(hostConfig),
+            };
+
+            // Dynamically load any additional registered extensions
+            // TODO: Determine the registration model (where we get candidate assemblies)
+            string[] extensionAssemblies = new string[]
+            {
+                "Microsoft.Azure.WebJobs.Extensions.ApiHub",
```

Comment:
This dynamic load code demonstrates that extensions can be completely decoupled. I actually removed the assembly references to all these assemblies from the project to prove it out. However, we'll probably add those back, and onboard these "built-ins" as we do above in a one liner.